### ISSUE TITLE
Few warnings

### ISSUE THREAD

**Issue by sibelius** [nanohop/sketch-to-react-native#2]
```
gathering elements...
(node:15370) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Evaluation failed: DOMException: Failed to execute 'querySelector' on 'Document': '#21993399543' is not a valid selector.
    at selector (<anonymous>:2:30)
    at <anonymous>:7:5
(node:15370) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

It does look promising!

**Comment by chrisachard** [comment]
@sibelius Ah, I think I know what's going on - would it be possible for me to get the sketch file you were trying to convert?  Either here or in an email?

**Comment by sibelius** [comment]
@chrisachard what's your email?

**Comment by chrisachard** [comment]
@sibelius chris@nanohop.com

**Comment by pablostanley** [comment]
I had the same problem 👍

**Comment by chrisachard** [comment]
@pablostanley This should be fixed in master (although fixing it revealed some other bugs...) - did you have the error recently?  If so, could you post or send your sketch file?  Thanks!

**Comment by chrisachard** [comment]
Fixed in a75b0f1e5be7d3cab1d20eec64bd1510613e0f56
